source/agora/consensus/protocol/Nominator.d

```diff
@@ -159,6 +157,42 @@ extern(D):
         return scp_quorum;
     }
 
+    /***************************************************************************
+
+        Set up the clients to which we'll be exchanging consensus messages with.
+
+        Params:
+            clients = the set of all clients we're networked with. A subset
+                      of the clients are in our quorum set, these will be
+                      the clients we exchange SCP messages with.
+
+    ***************************************************************************/
+
+    public void setupNetwork (NetworkClient[PublicKey] clients)
+    {
+        import agora.common.Set;
+        import std.algorithm;
+        import std.array;
+        import std.typecons;
+
+        void getNodes (QuorumConfig conf, ref bool[PublicKey] nodes)
+        {
+            foreach (node; conf.nodes)
+                nodes[node] = true;
+
+            foreach (sub_conf; conf.quorums)
+                getNodes(sub_conf, nodes);
+        }
+
+        // can't use Set(), requires serialization support
```

Comment:
Aha, I've just noticed this is no longer true! Yay refactorings

Reply:
And yay self reviews!